src/ring_theory/localization.lean

```diff
@@ -1335,6 +1382,21 @@ noncomputable def localization_algebra (M : submonoid R) (f : localization_map M
 variables (f : localization_map M Rₘ)
 variables (g : localization_map (algebra.algebra_map_submonoid S M) Sₘ)
 
+/-- Injectivity of the underlying `algebra_map` descends to the algebra induced by localization -/
+lemma localization_map_injective (hRS : function.injective (algebra_map R S))
```

Comment:
The name is a bit confusing with `localization_map`, because it suggests the conclusion is that `f` or `g` is injective.
```suggestion
lemma localization_algebra_injective (hRS : function.injective (algebra_map R S))
```